bindata/network/openshift-sdn/sdn-ovs.yaml

```diff
@@ -132,7 +132,14 @@ spec:
         lifecycle:
           preStop:
             exec:
-              command: ["/usr/share/openvswitch/scripts/ovs-ctl", "stop"]
+              command: 
+              - /bin/bash
+              - -c
+              - |
+                #!/bin/bash
+                /usr/share/openvswitch/scripts/ovs-ctl stop
+                rm -rf /run/openvswitch/*
+                rm -rf /var/lib/openvswitch/*
```

Comment:
We don't want to do this on ordinary termination; it would mean that every upgrade would tear down the network. (Likewise for the ovn-kube change.)

Reply:
fixed.